src/app/components/SubjectHeading/SubjectHeading.jsx

```diff
@@ -120,6 +105,50 @@ class SubjectHeading extends React.Component {
     this.context.router.push(`${path}/subject_headings/${this.props.subjectHeading.uuid}`)
   }
 
+  updateSortBy(sortByValue) {
+    if (this.state.sortBy !== sortByValue) {
+      this.state.sortBy = sortByValue;
+      this.state.range = new Range(0, Infinity, [{ start: 0, end: Infinity }]);
```

Comment:
Could this be done in `setState`?

Reply:
we don't actually want to re-render until after fetching the new data.